db/migrate/20230705014324_create_ingredients.rb

```diff
@@ -0,0 +1,18 @@
+class CreateIngredients < ActiveRecord::Migration[7.0]
+  def change
+    create_table :ingredients do |t|
+      t.references :recipe, null: false, foreign_key: true
+      t.string :name
+      t.integer :calories
+      t.integer :portion_size
+      t.string :category
+      t.float :fats
+      t.float :proteins
+      t.float :carbs
+      t.float :fibre
+      t.integer :sodium
```

Comment:
this is also float

Reply:
sodium's unit is mg. I check some foods. It is always integer